electrum/wallet.py

```diff
@@ -1747,19 +1747,26 @@ def fee_estimator(size: Union[int, float, Decimal]) -> int:
                     lower_bound = lower_bound if not is_local else 0
                     return int(max(lower_bound, original_fee_estimator(size)))
                 txi = base_tx.inputs()
-                txo = list(filter(lambda o: not self.is_change(o.address), base_tx.outputs()))
+                txo = list(filter(lambda o: not self.is_change(o.address), base_tx.outputs())) + list(outputs)
+                output_dict = {}
+                for output in txo:
+                    if output.address in output_dict:
+                        output_dict[output.address].value += output.value
+                    else:
+                        output_dict[output.address] = copy.copy(output)
+                txo = list(output_dict.values())
```

Comment:
This whole code block could be factored out to a generic function, moved to `transaction.py`: e.g.
```
def merge_tx_outputs(outputs1, outputs2)
```

Reply:
> ```
> def merge_tx_outputs(outputs1, outputs2)
> ```
@SomberNight, why do we need two arguments in the function?

Reply:
> This whole code block could be factored out to a generic function, moved to `transaction.py`: e.g.
> 
> ```
> def merge_tx_outputs(outputs1, outputs2)
> ```

@SomberNight, like this https://github.com/spesmilo/electrum/pull/8474/commits/f5a58675f131140de60c38a43194bdd5d6fdde1d?